test/core/end2end/generate_tests.bzl

```diff
@@ -401,12 +410,25 @@ def grpc_end2end_tests():
                 "//:grpc",
                 "//:gpr",
             ],
-            tags = ["no_windows"],
+            tags = _platform_support_tags(fopt),
         )
+
         for t, topt in END2END_TESTS.items():
             #print(_compatible(fopt, topt), f, t, fopt, topt)
             if not _compatible(fopt, topt):
                 continue
+
+            native.sh_test(
+                name = "%s_test@%s" % (f, t),
```

Comment:
For the sake of readability, can we change these names to something besides `f` and `t`?

Reply:
fair enough, but can we do as a followup to reduce the number of moving pieces in this PR?